### ISSUE TITLE
Full text search

### ISSUE THREAD

**Issue by Korb** [vit1251/golden#192]
I did not find any mention of the presence of full-text search of messages in the application in `README.md`. If it is implemented in the application, I suggest adding it to the list of functions supported by the application. At the same time, clarifying whether it supports regular expressions, fuzzy-, and morphology (indicating the supported languages).

If it is missing completely or partially, I suggest discussing the possibility of adding it.

**Comment by vit1251** [comment]
As far as I remember, there is no option for full-text search in the interface, and consequently, there are no mentions of it in the documentation.

Yes, let's discuss this. I am not planning to add such functionality for now, as I do not see a need for it. Are you interested in adding this feature?

**Comment by Korb** [comment]
> Are you interested in adding this feature?

Of course. It's hard for me to imagine a user-friendly communication tool without a search. Especially with an archive of communication of thousands of people over decades.

However, I'm not a programmer, and I can't help with the implementation of this functionality. At most, as a manual tester.

**Comment by vit1251** [comment]
>  I'm not a programmer, and I can't help with the implementation of this functionality.

This will make it very difficult to achieve desired goal of making the tool user-friendly. In any case, all I can recommend to you is to wait until someone wants to implement it, or at least suggest an approach for implementing full-text search.

I have not yet planned to allocate additional resources to continue the development of Golden Point.

>  At most, as a manual tester.

Yes, any help would be welcome.
